utils/gyb_syntax_support/__init__.py

```diff
@@ -179,3 +180,38 @@ def _digest_trivia(trivia):
         _digest_trivia(trivia)
 
     return digest.hexdigest()
+
+
+def syntax_collection_element_to_collection_mapping():
+    """
+    Generates a lookup table to find syntax collection type by syntax collection
+    element type.
+
+    Example:
+        {
+            'SyntaxBuildable': ['StringLiteralSegments', 
+                                'PrecedenceGroupAttributeList', 
+                                'AttributeList', 
+                                'SpecializeAttributeSpecList', 
+                                'SwitchCaseList'],
+	        'IfConfigClause': ['IfConfigClauseList'],
```

Comment:
I think this element is indented a little too much.

Reply:
Something like this?